packages/core/src/queries/log.ts

```diff
@@ -44,3 +44,17 @@ export const findLogs = async (limit: number, offset: number, logCondition: LogC
     limit ${limit}
     offset ${offset}
   `);
+
+export const getDnuCountsByTimeInterval = async (
+  startTimeInclusive: number,
+  endTimeExclusive: number
+) =>
+  envSet.pool.any<{ date: string; count: number }>(sql`
+    select date(${fields.createdAt}), count(*)
+    from ${table}
+    where ${fields.createdAt} >= to_timestamp(${startTimeInclusive}::double precision / 1000)
+    and ${fields.createdAt} < to_timestamp(${endTimeExclusive}::double precision / 1000)
+    and ${fields.type} in (${sql.join(registerLogTypes, sql`,`)})
```

Comment:
why not pass the log type in? So it can be used to filter other logs as well.

Reply:
Because `getDnuCountsByTimeInterval` was not intended to reuse this complex sql query, just inline the log types needed here. And there are no other places that can reuse `getDnuCountsByTimeInterval` (if pass the log types in) for now.